CMakeLists.txt

```diff
@@ -674,9 +674,7 @@ dolphin_find_optional_system_library_pkgconfig(ZSTD libzstd>=1.4.0 zstd::zstd Ex
 
 dolphin_find_optional_system_library_pkgconfig(ZLIB zlib-ng ZLIB::ZLIB Externals/zlib-ng)
 
-# https://github.com/zlib-ng/minizip-ng/commit/6c5f265a55f1a12a7a016cd2962feff91cff5d2e
-add_definitions(-DMZ_COMPAT_VERSION=110)  # This macro is for forwards compatibility with 4.0.4+
-dolphin_find_optional_system_library_pkgconfig(MINIZIP minizip>=3.0.0 minizip::minizip Externals/minizip)
+dolphin_find_optional_system_library_pkgconfig(MINIZIP minizip-ng>=4.0.4 minizip::minizip Externals/minizip-ng)
```

Comment:
Whether there's the `-ng` suffix or not is completely decided by the distro: https://github.com/zlib-ng/minizip-ng/blob/5aec551bf78ed79f6bec3160a5c7ce7fdac66776/CMakeLists.txt#L678
Arch and NixOS for instance do add the suffix, but I don't know if there's a distro that doesn't.

Reply:
We already compile a stripped-down version of the library when statically linking (see my added documentation on unused files in Dolphin's CMakeLists.txt for it), so I think it's not a huge loss if some distros handle it weirdly. Thank you for the input, though.

Reply:
I have added a comment to our CMakeLists to document this discrepancy in case anyone in the future with one of these unknown distros attempts to switch it back to "minizip".

Reply:
OpenBSD doesn't call the new version -ng: https://cvsweb.openbsd.org/cgi-bin/cvsweb/ports/archivers/minizip/pkg/PLIST?rev=1.8&content-type=text/x-cvsweb-markup